src/components/Card.astro

```diff
@@ -20,40 +20,138 @@ const {
 } = Astro.props
 ---
 
-<article
-	class="duration-50 relative z-0 flex h-full flex-col gap-4 overflow-hidden rounded-lg border border-neutral-200 bg-white p-4 text-astro-gray-700 shadow shadow-neutral-200 transition-colors hover:border-primary-300 hover:shadow-accent-200"
->
-	<header class="flex flex-row flex-wrap justify-between gap-2">
-		<h3 class="text-lg font-bold">{title}</h3>
-		<a
-			class="ease scale-90 opacity-60 transition-all duration-100 hover:scale-100 hover:opacity-100"
-			href={sourceUrl}
-			aria-label="View Source Code"
+<article class="relative border border-astro-gray-500 bg-astro-gray-600">
+	<div class="noise z-0"></div>
+	<div class="aspect-video bg-astro-gray-700">
+		{
+			!!previewImage && (
+				<img src={previewImage} alt="" class="object-cover s-full" />
+			)
+		}
+	</div>
+	<div class="flex flex-col gap-4 p-5">
+		<h3 class="heading-4">{title}</h3>
+		<p>
+			<a href={sourceUrl} class="flex items-center gap-1">
+				<Icon name="github" size={16} />
+				<span
+					class="text-sm text-astro-gray-200 underline underline-offset-[3px]"
+					>View repo on GitHub</span
```

Comment:
<img width="413" alt="Screenshot 2023-04-04 at 4 19 37 PM" src="https://user-images.githubusercontent.com/15836226/229925386-0aee17cb-6e86-41fe-842e-e6733d2c7670.png">

tiny nit: this could use a flex align to avoid the full-width hit target + a bit of vertical padding inside the `<a>` for a larger click target